src/sbt-test/sbt-scalac-opts-plugin/scalacOptions/build.sbt

```diff
@@ -1,3 +1,10 @@
 crossScalaVersions := Seq(
   "2.12.8",
-  "2.13.0-RC1")
+  "2.12.9",
+  "2.12.10",
+  "2.12.11",
+  "2.13.0",
+  "2.13.1",
+  "2.13.2",
+  "2.13.3"
```

Comment:
why do we need so many cross versions?

Reply:
that is for internal test project, but I can leave only latest two

Reply:
yes please,  either last two or convince that "your use case" is usable for others as well :)

Reply:
I experimented and expected build failure on 2.13.3 due to `-Xlint`, but it didn't happen, only error message in log, hard to spot - removed useless targs